acceptance/suites/tests/full/20_client_commands/configuration_settings/C61687_authenticate_with_wrong_format_of_credentials.rb

```diff
@@ -28,7 +28,7 @@
 
     step "Set up users on #{agent}"
     on(agent, 'cat /etc/puppetlabs/razor-server/shiro.ini') do |result|
-      assert_match /^\s*razor = razor/, result.stdout, 'User razor should already have password "razor"'
+      assert_match /^\s*razor = 9b4f1d0e11dcc029c3493d945e44ee077b68978466c0aab6d1ce453aac5f0384/, result.stdout, 'User razor should already have password "razor"'
```

Comment:
Might be worth filling a ticket to clean-up some duplicate code in the tests. We have this same set-up of enabling auth. then setting up our shiro-ini, asserting that the razor user is created, in 9 tests.